packages/ckeditor5-list/src/listproperties/utils/config.ts

```diff
@@ -28,12 +31,33 @@ import { type ListType } from '../../list/listediting.js';
  */
 export function getNormalizedConfig( config: ListPropertiesConfig ): NormalizedListPropertiesConfig {
 	const { startIndex, reversed, styles } = config;
-	const normalizedConfig = {
-		listTypes: [ 'bulleted', 'numbered' ] as Array<ListType>,
-		useAttribute: false,
+
+	return {
+		styles: getNormalizedStyles( styles ),
 		startIndex: startIndex || false,
 		reversed: reversed || false
 	};
+}
+
+/**
+ * Normalizes styles in the configuration of the list properties feature.
+ * The structure of normalized list properties options looks as follows:
+ *
+ * ```ts
+ * {
+ * 	listTypes: [ 'bulleted', 'numbered' ],
+ * 	useAttribute: false
+ * }
+ * ```
+ *
+ * @param config The list properties styles.
+ * @returns An object with normalized list properties styles.
+ */
+export function getNormalizedStyles( styles: ListPropertiesStyles ): NormalizedListPropertiesStyles {
```

Comment:
I'm not sure if we need this helper exported. I'd suggest using it only internally in this file.
```suggestion
function getNormalizedStylesConfig( styles: ListPropertiesStyles ): NormalizedListPropertiesStyles {
```